common/src/main/java/eu/midnightdust/lib/config/MidnightConfig.java

```diff
@@ -85,6 +85,25 @@ public <T> void writeList(int index, T value) {
         } return null;
     }
 
+    public static boolean checkRequirements(Requires req) {
+        String reqString = req.requirement();
+        Field reqField;
+        try {
+             reqField = req.requirementSource().getDeclaredField(reqString);
+        } catch (NoSuchFieldException e) {
+            throw new RuntimeException("Field does not exist!", e);
+        }
+
+        reqField.setAccessible(true);
+
+        try {
+            return reqField.getBoolean(null);
+        } catch (IllegalAccessException e) {
+            throw new RuntimeException("Field is not accessible!", e);
```

Comment:
Same here :)

Reply:
We could also simplify both try/catch blocks into just one by specifying `catch(NoSuchFieldException | IllegalAccessException e)` or even simpler (which would probably even reduce file size by a tiny bit) `catch(Exception e)`